docs/server-request.md

```diff
@@ -17,20 +17,38 @@ A low-level action that allows for any kind of HTTP request to be performed.
 - Read files or binary streams
 - Call APIs from same app system - use the [Call Local API](/docs/Actions/CallLocalApi.md) action instead
 
-## `Related Actions`
+## `Related actions`
 
 - [Parse JSON](/docs/Actions/CallApi.md) - creates variables from JSON returned by API calls
 - [Regex](/docs/Actions/CallApi.md) - create variables by matching patterns inside the content
 
-## `Parameter Reference`
+## `Parameter reference`
 
-| Parameter     | Description                           | Supports Variables |
+| Parameter     | Description                           | Supports variables |
```

Comment:
Maybe add **Default Value** and **Required** colmuns.
```suggestion
| Parameter     | Description                           |Default Value |Required | Supports variables |
```

Also keep all the settings in the same place. Add checkboxes also here basically

Reply:
Maybe call this section Input parameters for consistency, since the other is Output parameters